packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -110,7 +110,8 @@ const ListSectionHeader = ({
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
   }
-
+  /* eslint-disable-next-line */
```

Comment:
what are you disabling here

Reply:
Due to no-param-reassign error i have added this.